ngx_http_sxg_filter_module.c

```diff
@@ -369,7 +369,12 @@ static ngx_int_t subresource_fetch_handler_impl(ngx_http_request_t* req,
     --ctx->subresources;
     return NGX_OK;
   }
-  if (req->out->buf->last - req->out->buf->pos == req->upstream->length) {
+  ngx_log_error(NGX_LOG_DEBUG, req->connection->log, 0,
+                "nginx-sxg-module: Subresource fetched. bytes=%l/%l",
+                req->out->buf->last - req->out->buf->pos,
+                req->upstream->length);
+  if (req->out->buf->last - req->out->buf->pos == req->upstream->length ||
+      req->upstream->length == -1) {
```

Comment:
Can you put a short description comment for the case `upstream->length == -1` (maybe failed to load?)

Reply:
I added the description.